CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# v0.2.2 dbt_qualtrics_source
+[PR #9](https://github.com/fivetran/dbt_qualtrics_source/pull/9) includes the following updates: 
+
+## Bug Fix
+- Updated unique combination of columns test on `stg_qualtrics__distribution_contact` to include `contact_lookup_id`. [PR #8](https://github.com/fivetran/dbt_qualtrics_source/pull/8)
```

Comment:
I would add a reason why it had to be added (was it accidentally omitted or was there a connector update, etc?)

Reply:
Updated